mllib/src/test/scala/org/apache/spark/ml/evaluation/RegressionEvaluatorSuite.scala

```diff
@@ -65,15 +65,15 @@ class RegressionEvaluatorSuite
 
     // default = rmse
     val evaluator = new RegressionEvaluator()
-    assert(evaluator.evaluate(predictions) ~== 0.1019382 absTol 0.001)
+    assert(evaluator.evaluate(predictions) ~== 0.1013829 absTol 0.001)
```

Comment:
Yeah I expect some values change due to differing use of the RNG. Some of the differences are larger than the test's tolerance. That could just mean the tolerance is too tight. I wonder if it's worth loosening them where it's clearly too small (implicitly, by adjusting the expected value we're accepting this). Like, these could be +/- 0.01.